.tekton/tasks/ec-checks.yaml

```
@@ -25,34 +25,28 @@ spec:
   - name: validate-all-tasks
     workingDir: "$(workspaces.source.path)/source"
     image: quay.io/enterprise-contract/ec-cli:snapshot
-    command: [ec]
-    args:
-      - validate
-      - definition
-      - "--file"
-      - "./all_tasks-ec"
-      - "--policy"
-      - "git::https://github.com/enterprise-contract/ec-policies//policy/task"
-      - "--policy"
-      - "git::https://github.com/enterprise-contract/ec-policies//policy/lib"
-      - "--data"
-      - "git::https://github.com/release-engineering/rhtap-ec-policy//data"
-      - "--strict"
+    script: |
+      set -euo pipefail
+
+      # Generate list of file parameters, e.g. --file=foo.yaml --file=bar.yaml
+      files=$(ls all_tasks-ec/*.yaml | awk '{printf " --file=" $0}')
+      echo "[DEBUG] Files parameter: ${files}"
+
+      policy='enterprise-contract-service/redhat-trusted-tasks'
+
+      ec validate input --policy "${policy}" --output yaml --strict=true ${files}
```

This can be done without shelling out to external commands:
```suggestion
      files=(all_tasks-ec/*.yaml)
      args=${files[*]/#/--file }
      echo "[DEBUG] Files parameter: ${args[*]}"

      policy=enterprise-contract-service/redhat-trusted-tasks

      ec validate input --policy "${policy}" --output yaml --strict=true ${args[*]}
```

---

That is fantastic. I'll incorporate that approach.

---

Done.